app/views/shared/_footer.html.erb

```diff
@@ -7,15 +7,5 @@
         <li class="list-inline-item mr-3"><%= link_to "Privacy", privacy_path %></li>
       </ul>
     </div>
-    <div class=" pull-right">
-      <ul class="navbar-nav mb-0 ml-3 flex-row">
-        <li class="nav-item px-2"><a href="https://github.com/milspousecoders"><i class="fa fa-github"></i></a></li>
-        <li class="nav-item px-2"><a href="https://www.facebook.com/MilSpouseCoders"><i class="fa fa-facebook"></i></a></li>
-        <li class="nav-item px-2"><a href="https://www.instagram.com/milspousecoders/"><i class="fa fa-instagram"></i></a></li>
-        <li class="nav-item px-2"><a href="https://twitter.com/MILSpouseCoders"><i class="fa fa-twitter"></i></a></li>
-        <li class="nav-item px-2"><a href="https://plus.google.com/115988512421332416550/posts"><i class="fa fa-google-plus"></i> </a></li>
-        <li class="nav-item px-2"><a href="https://www.youtube.com/channel/UCu0hl34X3ko53WSbHI1Olkw"><i class="fa fa-youtube"></i></a></li>
-      </ul>
-    </div>
```

Comment:
Good call - the issue you created will be great for someone who is just starting to learn HTML & CSS.